src/components/NotFoundController.jsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * NotFoundController simply renders the NotFound component. This controller is
+ * necessary for transitions. Without it, the component will not recieve the
+ * function visibleTransitionOut and the page will transition will fail.
+ */
+
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController';
+
+// Components
+import NotFound from 'components/NotFound';
+
+export default class NotFoundController extends FalcorController {
+  static getFalcorPathSets() {
```

Comment:
Is there a reason this needs to be a FalcorController?

By design, FalcorControllers must be included in the routes.js map, I think. Otherwise they are not guaranteed to server side render correctly, because they willl not fetch their data. This element is not in the routes map.